package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@slice-and-dice/register-a-food-business-validation",
-  "version": "1.21.1",
+  "version": "1.21.2",
```

Comment:
This needs to be 1.21.3 as there was change merged between your last one and this one.  Also change changelog. 
Probably worth merging from master into your branch to get the latest changes and check for conflicts.